Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -765,6 +822,16 @@ def tc_get_indicator_command():
                                              headerTransform=pascalToSpace)
         })
 
+    if include_attributes:
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': indicator_attributes,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('ThreatConnect Attributes for indicator: {}'.format(indicator), indicator_attributes,
+                                             headerTransform=pascalToSpace)
+        })
+
```

Comment:
Provide the results from extracting the Indicator Attributes as a markdown table